packages/commonwealth/server/routes/bulkOffchain.ts

```diff
@@ -120,6 +123,10 @@ const bulkOffchain = async (models: DB, req: Request, res: Response) => {
         type: QueryTypes.SELECT,
       }
     ),
+    await models.sequelize.query(bulkThreadsQueryWithCutoffDate(true, true), {
```

Comment:
I worry that this query is too heavyweight for the use case, even with the ternary on count(), as we don't need to perform all the joins when we're just querying the number of threads available. It may be better to have a small, separate query here instead of trying to reuse the bulkThreads query.

Was there a particular reason you did this refactor?

Reply:
I agree the query is heavy for the usecase, i used just to play safe :sweat_smile:. Will be adding a shorter query for it

Reply:
Fixed [here](https://github.com/hicommonwealth/commonwealth/pull/3534/commits/7a37eff5f5035654921c00235a1ffd053e6cfd63)